src/app.rs

```diff
@@ -34,13 +34,21 @@ impl App {
     pub fn render<B: Backend>(&mut self, frame: &mut Frame<'_, B>) {
         // This is where you add new widgets.
         // See the following resources:
-        // - https://docs.rs/tui/0.16.0/tui/widgets/index.html
-        // - https://github.com/fdehau/tui-rs/tree/v0.16.0/examples
+        // - https://docs.rs/tui/0.19.0/tui/widgets/index.html
```

Comment:
Maybe using `latest` makes more sense here so that we won't update this comment with every new version of tui? e.g. `https://docs.rs/tui/latest/tui/widgets/index.html`